hw/ip/clkmgr/doc/dv/index.md

```diff
@@ -33,34 +25,31 @@ CLKMGR testbench has been constructed based on the [CIP testbench architecture](
 ![Block diagram](tb.svg)
 
 ### Top level testbench
-Top level testbench is located at `hw/ip/clkmgr/dv/tb.sv`. It instantiates the CLKMGR DUT module `hw/ip/clkmgr/rtl/clkmgr.sv`.
+Top level testbench is located at `hw/ip/clkmgr/dv/tb.sv`. It instantiates the CLKMGR DUT module `hw/top_earlgrey/ip/clkmgr/rtl/autogen/clkmgr.sv`.
 In addition, it instantiates the following interfaces, connects them to the DUT and sets their handle into `uvm_config_db`:
 * [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs" >}})
 * [TileLink host interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
-* CLKMGR IOs
+* CLKMGR IOs: `hw/ip/clkmgr/dv/env/clkmgr_if.sv`
 * Interrupts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
 * Alerts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
```

Comment:
Would you mind help changing this line to here:
`Alerts ([alert_esc_if]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}))`

Reply:
Done.